0_intro/exercises/factorial/src/lib.rs

```diff
@@ -1,5 +1,9 @@
 pub fn factorial(n: i32) -> i32 {
-    todo!()
+    if n == 0 {
+        1
+    } else {
+        n * factorial(n - 1)
+    }
```

Comment:
Also don't you think this would look cool with a match statement instead of an `if`? :P